deployment/aws-terraform/README.md

```diff
@@ -0,0 +1,40 @@
+# Cloud One File Storage Security Terraform Deployment
+
+Cloud One File Storage Security stacks deployment by Terraform.
+
+## Prerequisites
+
+1. The [Terraform CLI](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started) (0.14.9+) installed.
```

Comment:
1. The [Terraform CLI](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started) (0.14.9+) is installed.